src/main/java/com/epam/emotionalhelp/controller/QuestionController.java

```diff
@@ -42,15 +46,23 @@ public QuestionResponseDto findById(@PathVariable Long id) {
 
     @ResponseStatus(HttpStatus.CREATED)
     @PostMapping
-    public QuestionResponseDto create(@RequestBody QuestionRequestDto questionRequestDto) {
-        return questionService.create(questionRequestDto);
+    public ResponseEntity<Object> add(@RequestBody @Valid QuestionRequestDto questionRequestDto) {
+        Question question = questionService.addQuestion(questionRequestDto);
+        return ResponseHandler.
+                generateResponse(ResponseMessage.SUCCESSFULLY_CREATED,
+                        HttpStatus.CREATED,
+                        QuestionMapper.toDto(question));
     }
 
-    @PatchMapping("/{id}")
-    public QuestionResponseDto update(@PathVariable Long id, @RequestBody QuestionRequestDto questionRequestDto) {
-        return questionService.update(id, questionRequestDto);
+    @PutMapping("/{id}")
+    public ResponseEntity<Object> update(@PathVariable Long id,
```

Comment:
Just a proposal:
can we add some validation to path variable like `@Min(MIN_ID_VALUE) @PathVariable Long id` ?